src/libraries/System.Text.Json/src/System/Text/Json/Serialization/WriteStackFrame.cs

```diff
@@ -120,5 +120,8 @@ public JsonConverter InitializeReEntry(Type type, JsonSerializerOptions options)
 
             return PolymorphicJsonPropertyInfo.ConverterBase;
         }
+
+        [DebuggerBrowsable(DebuggerBrowsableState.Never)]
+        private string DebuggerDisplay => $"ConverterStrategy.{JsonTypeInfo?.PropertyInfoForTypeInfo.ConverterStrategy}, {JsonTypeInfo?.Type.Name}";
```

Comment:
In some cases we read from `JsonPropertyInfo.ConverterStrategy` while in others, we read from `JsonPropertyInfo.ConverterBase.ConverterStrategy`. Could this be made consistent?